tasks/main.yml

```diff
@@ -125,6 +125,8 @@
     repo: "{{ grafana_conf_grafana_dashboard_repo.repo }}"
     dest: "{{ grafana_conf_grafana_dashboard_repo.checkout_path }}"
     version: "{{ grafana_conf_grafana_dashboard_repo.version | default('HEAD') }}"
+    accept_hostkey: yes
```

Comment:
Isn't not checking the host key bad practice?

Reply:
Yep, I have removed it.